GOVERNANCE.md

```diff
@@ -166,19 +166,18 @@ Provide a summary of the nominee's contributions. For example:
   organization
 * Other participation in the wider Node.js community
 
-Mention @nodejs/collaborators in the issue to notify other collaborators about
-the nomination.
+If the private discussion receives some support (in form of explicit comments or
+thumbs-up reactions) and no explicit block after a few days, add a comment in the
```

Comment:
If we can clarify here what constitutes an "explicit block" and handle the self-nomination loophole then I'm ok with the rest of these changes.

I don't believe a mandatory private nomination while allowing for self public nominations are compatible with each other.

Reply:
> I don't believe a mandatory private nomination while allowing for self public nominations are compatible with each other.

Why? The private nomination is to protect the nominee – because they, and the rest of the world, do not need to know they're being rejected if they haven't asked for anything. In the case of a self-nomination, we cannot hide from them whether they're being accepted, and they open themself to the risk of being rejected.

Reply:
Because it can be easily gamed. If an existing collaborator does not want to follow the formal process, they can simply ask the person to nominate themselves and sidestep the formal process, causing issues for both other collaborators and the nominee who may not understand the possible risks. If we are making the private process mandatory then let's make it mandatory in all cases and we avoid this possibility.

Reply:
I’ll make another pass to the text to make the private discussion optional but strongly recommended, hopefully that would address this concern (FWIW I’m a -1 on removing self-nominations, having a private-only process could enable (not sure if it’s the right word) discriminatory gatekeeping)

Reply:
I've decided to not try to define what a block is, I think the informal thing is enough given it's an optional step anyway. I'd expect the "I'm going to open the public issue in 72 hours" be enough to let folks who are actually blocking to react in time in case their blocking comment wasn't explicit enough